src/views/OnboardingSelector.vue

```diff
@@ -7,7 +7,7 @@
 
             <button v-if="!request.disableBack" class="global-close nq-button-s" @click="close">
                 <ArrowLeftSmallIcon/>
-                Back to {{request.appName}}
+                Back to {{isCheckoutOnboarding ? 'Checkout' : (isChooseAddressOnboarding ? 'Choose Address' : request.appName)}}
```

Comment:
Now with two conditions, this should be moved to a separate computed prop, like `backButtonText` that does to computing in the class instead of in the template.